handlers/company_handler.py

```
@@ -0,0 +1,51 @@
+from fastapi import HTTPException
+from dtos.request.company_req_dto import CreateCompanyReqDTO, ChangeCompanyReqDTO
+from dtos.response.company_res_dto import CompanyResDTO, CompanyListResDTO
+from usecases.company_usecase import CompanyUseCase
+
+
+class CompanyHandler:
+    def create_company(self, company_req: CreateCompanyReqDTO) -> CompanyResDTO:
+        """회사 생성"""
+        try:
+            return CompanyUseCase.create_company(company_req)
+        except HTTPException as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)